src/biogeochem/DUSTMod.F90

```diff
@@ -30,6 +30,8 @@ module DUSTMod
   use LandunitType         , only : lun
   use ColumnType           , only : col
   use PatchType            , only : patch
+  use ZenderSoilErodStreamType,  only : zendersoilerodstream_type
```

Comment:
This name is long and does not use snake_case. Erik will decide whether to keep the "zender" prefix to make it obvious that only the zender method uses it or remove because theoretically other methods could use it.

Reply:
Went with snake_case and removed zender prefix.